### ISSUE TITLE
RadioGroup - fix accessibility violations

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/B01LER-Kitchen#775]
#### Description / User Story

The following errors and incomplete tests are thrown by the storybook accessibility addon and should be fixed:
- Incomplete: ARIA attributes must conform to valid values (disabled=true / hasError=true)

----
#### Requirements
- [ ] #536  

----
#### Acceptance Criteria
- [ ] All errors and incomplete tests thrown by the accessibility addon are resolved:
   - [ ] Incomplete: ARIA attributes must conform to valid values (disabled=true / hasError=true)

----
#### Background information
- ...
